stea/stea_config.py

```diff
@@ -103,19 +96,19 @@ def _build_schema():
                                     "data, but you can optionally use the keywords start-year and end-year to"
                                     "limit the time range."
                                 ),
-                                MK.Required: False,
+                                MK.AllowNone: True,
                             },
                             "mult": {
                                 MK.Type: types.List,
-                                MK.Required: False,
                                 MK.Description: "List of multipliers of eclsum key",
-                                MK.Content: {MK.Item: {MK.Type: types.Number}},
+                                MK.Content: {
+                                    MK.Item: {MK.Type: types.Number, MK.AllowNone: True}
+                                },
                             },
                             "glob_mult": {
                                 MK.Type: types.Number,
-                                MK.Required: False,
                                 MK.Description: "A single global multiplier of eclsum key",
-                                MK.Content: {MK.Item: {MK.Type: types.Integer}},
```

Comment:
Not quite sure about this one, why does a number have content?

Reply:
Not sure, but are we also here changing it to only accept an integer, to now accept e.g. floats? (an to be honest, why shouldn't a multiplier be able to be a float?)

Reply:
`Content` on a number doesn't make sense to me either

Reply:
I don't think `Content` for any basic types makes any sense

Reply:
I agree that it shouldn't be content, but we should be certain that the type is correct.

Reply:
Here is where it is used:
https://github.com/equinor/stea/blob/a8392009a40f8ba390906ee631869a703b97e8f7/stea/stea_request.py#L82

So it's not going to the backend. We won't be breaking any of the current configurations either by doing this, should be okay.

Reply:
>Is that a configsuite bug?
Am I missing something by the way? Is it the lack of error you are considering to be a bug? Because it does work without the `MK.Content`..

The lack of error I agree could be a bug. I don't think it should be possible to add `Content` to any BasicTypes either

Reply:
`I don't think it should be possible to add Content to any BasicTypes either` that is what I was thinking of.

Reply:
This is interesting, because the `Content` is definitively unnecessary and should be removed. However, when an element is validated the element is validated first, ignoring `Content` and then it is validated recursively afterwards if a container. Hence, it will not complain about a `Content` element for a basic type....

I'll write up an issue

Reply:
https://github.com/equinor/configsuite/issues/146